images/aws-scripts/deploy-kubeflow.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This shell script is used to deploy kubeflow by kfctl
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+EKS_CLUSTER_NAME="${EKS_CLUSTER_NAME}"
+
+cd /tmp
+
+# Fetch v1.1-branch kfctl
+wget https://github.com/kubeflow/kfctl/releases/download/v1.1.0/kfctl_v1.1.0-0-g9a3621e_linux.tar.gz -O kfctl.tar.gz
```

Comment:
the scripts will be used in which tests?  Seems we use fixed version here

Reply:
We need to maintain a compatibility table:
```
version  |  kfctl location            |  manifest location
v1.1.0   |  kfctl_v1.1.0-xxx.tar.gz   |  kfctl_aws_v1.1.0-patch.yaml 
```